components/table/table-wrapper.js

```diff
@@ -388,6 +424,10 @@ export class TableWrapper extends RtlMixin(PageableMixin(SelectionMixin(LitEleme
 		this._table = e.target.assignedNodes({ flatten: true }).find(
 			node => (node.nodeType === Node.ELEMENT_NODE && node.tagName === 'TABLE' && node.classList.contains('d2l-table'))
 		);
+		this._tableScrollers = (this.scrollWrapper && this.stickyHeaders) ? {
+			primary: this._table?.querySelector('tbody'),
+			secondary: this._table?.querySelector('thead'),
```

Comment:
Not sure if we care about this, but there's a weird edge case here where if they don't use `tbody` and `thead` but do set `scroll-wrapper`, then it'll try to use it but fail and result in just a normal scroll-wrapper... which isn't going to work very well.

Reply:
So I've just double-checked on our earliest supported browsers on Saucelabs, and they all automatically create a `tbody` when the table has any contents. The thead needs to be explicit, even on modern browsers, which makes sense.

So it'll always treat the tbody as the primary scroller, it just might not have any linked scroller if there's no thead. That sounds okay to me, since the thead is the only thing that can be sticky anyway...

I've updated the jsdoc to call out more explicitly that `sticky-headers-scroll-wrapper` requires the sticky headers to be in `thead`.

Reply:
Nice -- sounds good!